### PR TITLE
Uda rst

### PR DESCRIPTION
**Pull request by joakim-hove**
Load UDA properties from the restart file and assign to the well and group controls. ~~Group injection not yet correctly handled.~~

Some trivial commits are extracted here:  https://github.com/OPM/opm-common/pull/2632

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQActive.cpp [hunk 1]
```diff
@@ -16,15 +16,44 @@
...
+    for (const auto& record : rst_active.iuad) {
+        const auto& udq_input = udq_config[record.input_index];
+        UDAValue uda(udq_input.keyword(), units.uda_dim(record.control));
+        for (std::size_t use_index = 0; use_index < record.use_count; use_index++) {
+            std::size_t wg_index = rst_active.wg_index[record.wg_offset + use_index];
+
+            if (UDQ::well_control(record.control))
+                records.emplace_back(record.control, uda, well_names[wg_index]);
+            else {
+                if (UDQ::production_control)
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQActive.cpp:47]: This line (`if (UDQ::production_control)`) gives a [warning](https://ci.opm-project.org/job/opm-common-PR-builder/4097/gcc/fileName.-16558701/source.c47caf56-ce55-4457-a508-cafb4ef66cbd/#47) that the "address of `UDQ::production_control` will never be `NULL`.  Did you mean to pass an argument here?

Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQActive.cpp]: Thank you - yes you are right this should be called with an argument.

**Comment by joakim-hove** [comment]
jenkins build this please

**Pull request merged**